test/sqlsrv/srv_029_data_conversion_varchar_datetime.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+Data type precedence: conversion VARCHAR(n)
+--SKIPIF--
+--FILE--
+<?php
+
+require_once("autonomous_setup.php");
+
+// Connect
+$conn = sqlsrv_connect($serverName, $connectionInfo) ?: die();
+
+// Create database
+sqlsrv_query($conn,"CREATE DATABASE ". $dbName) ?: die();
```

Comment:
It seems a little over the top to create a database for one simple test... is it possible that you create a table with some sort of random number to guarantee its uniqueness?